jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -322,7 +322,7 @@ properties:
           msa_configuration_file:
             title: MSA configuration file name
```

Comment:
A minor nit, but let's also change the comment portion of the FITS keyword record to "MSA metadata file name". That's what's now in the keyword dictionary and is consistent with the way this file is now being called.